From 71e19d7aea44541a67ab6fea8baecfd35e05e3f4 Mon Sep 17 00:00:00 2001
From: Qi Yadong <yadong.qi@intel.com>
Date: Tue, 3 Sep 2019 15:17:48 +0800
Subject: [PATCH] [DO-NOT-MERGE]CFG for AIV

1. disable suspend support
2. disable VT-d
3. disable isolation for MSR_PERF_CTRL_ISOLATION

Change-Id: If25e377b86ac6a9d494e0fbcfe6bc295e1378c37
Signed-off-by: Qi Yadong <yadong.qi@intel.com>
---
 product/celadon_64.cfg        | 12 +++++++-----
 product/feature/isolation.cfg |  4 ++--
 2 files changed, 9 insertions(+), 7 deletions(-)

diff --git a/product/celadon_64.cfg b/product/celadon_64.cfg
index 8344298..caab540 100644
--- a/product/celadon_64.cfg
+++ b/product/celadon_64.cfg
@@ -6,7 +6,7 @@ include $(PROJS)/product/board/cel_nuc.cfg
 include $(PROJS)/product/feature/trusty_tee.cfg
 include $(PROJS)/product/feature/isolation.cfg
 include $(PROJS)/product/feature/security.cfg
-include $(PROJS)/product/feature/suspend.cfg
+#include $(PROJS)/product/feature/suspend.cfg
 
 export LOADER_STAGE0_SUB = efi
 
@@ -15,10 +15,12 @@ EVMM_CMPL_FLAGS += \
  -DMODULE_TSC \
  -DMODULE_VMEXIT_INIT\
  -DPACK_LK \
- -DMODULE_ACPI \
- -DMODULE_VTD \
- -DDMAR_MAX_ENGINE=16 \
- -DSKIP_DMAR_GPU
+ -DMODULE_ACPI
+
+#EVMM_CMPL_FLAGS += \
+# -DMODULE_VTD \
+# -DDMAR_MAX_ENGINE=16 \
+# -DSKIP_DMAR_GPU
 
 EVMM_CMPL_FLAGS += \
  -DAP_START_IN_HLT
diff --git a/product/feature/isolation.cfg b/product/feature/isolation.cfg
index 201093d..75ab345 100644
--- a/product/feature/isolation.cfg
+++ b/product/feature/isolation.cfg
@@ -1,7 +1,6 @@
 EVMM_CMPL_FLAGS += \
  -DMODULE_DR \
- -DMODULE_XSAVE \
- -DMODULE_PERF_CTRL_ISOLATION
+ -DMODULE_XSAVE
 
 #Isolate MSR_TSC/MSR_TSC_ADJUST
 EVMM_CMPL_FLAGS += \
@@ -16,4 +15,5 @@ EVMM_CMPL_FLAGS += \
 # not used modules
 #-DMODULE_FXSAVE \
 #-DMODULE_CR \
+#-DMODULE_PERF_CTRL_ISOLATION
 
-- 
2.23.0

